src/cosalib/cmdlib.py

```diff
@@ -177,7 +177,7 @@ def disk_ignition_version(path):
         return "3.0.0"
 
 
-def import_ostree_commit(repo, commit, tarfile):
+def import_ostree_commit(repo, commit, tarfile, force=False):
     # create repo in case e.g. tmp/ was cleared out; idempotent
     subprocess.check_call(['ostree', 'init', '--repo', repo, '--mode=archive'])
```

Comment:
it seems like this command should fail if there is a repo that exists already.

Reply:
Hmm, how so? See https://github.com/ostreedev/ostree/commit/5334758ba7309438686c779e4ff943a5f84b6868 for the background on this. Anyway, I don't think we can break compatibility on this now.

(Though the behaviour of ignoring existing repos of a different mode can definitely be misleading.)

Reply:
> Hmm, how so? See [ostreedev/ostree@5334758](https://github.com/ostreedev/ostree/commit/5334758ba7309438686c779e4ff943a5f84b6868) for the background on this.

I was surprised by the behavior. That's all. 

> Anyway, I don't think we can break compatibility on this now.

You're probably right.

> 
> (Though the behaviour of ignoring existing repos of a different mode can definitely be misleading.)

Agree.

Reply:
Gotcha, I thought you were suggesting changing it.